components/common/src/main/java/com/hotels/styx/common/CompletableFutures.java

```diff
@@ -27,14 +27,9 @@ private CompletableFutures() {
     }
 
     // TODO: We need to substitute this in favour of Styx APIs
-    //
-    // Credit: http://www.nurkiewicz.com/2014/11/converting-between-completablefuture.html
     public static <T> CompletableFuture<T> fromSingleObservable(Observable<T> observable) {
```

Comment:
If we keep the method in the API public, I don't know why we need this method at all.

Reply:
It should have been private.